### ISSUE TITLE
AutoMap seems to be case sensitive

### ISSUE THREAD

**Issue by kevinsaucier** [xLightsSequencer/xLights#5037]
Could be a Mac issue, since Linux is case sensitive but......

TLDR:
I have a mapping file.  I start a new sequence, I open the import dialog, I open my mapping file.

1. 'Spinner - Showstopper' (left) gets a mapping to 'SHOWSTOPPER SPINNER' (from a previous session saved into the mapping file).  No red background on the left.
2. 'Showstopper Spinner' (right) does not show as mapped.  No change to the highlight color.  Running the import does not import the effects.
3. Replace the mapping on the left with 'Showstopper Spinner' on the right changes the highlight color on the right and the import works as expected.  

AutoMap (the button) works as expected if you run it first but, if I then load my mapping file, it overwrites the automapped entries with the ALL CAPS entries, which doesn't work.  If I then AutoMap (button) again, it doesn't replace the ALL CAPS entries and importing the effects doesn't work.

I would expect the matching/import process to be case-insensitive.  

Example:

Here, if I open the mapping file first, you can see 23 Fireworks shows as mapped on the left but the right side is not dimmed.

![image](https://github.com/user-attachments/assets/ae6c7500-c112-4e39-9c0f-a153053b35c1)

Clicking AutoMap maps the other items but doesn't overwrite 23 Fireworks (ALL CAPS) on the left

![image](https://github.com/user-attachments/assets/e9b9a830-5123-44bb-9da2-9aa8145c9fff)

If I reopen the import and click AutoMap first, you can see all submodels automap correctly (dimmed on the right)

![image](https://github.com/user-attachments/assets/3b95890a-ef76-4266-bb99-d28df8131ed7)

If I then reload the mapping file, it undoes the AutoMap and it's back to the non-working maps.

![image](https://github.com/user-attachments/assets/d8f9476a-721b-4618-901d-f77aecc681f0)

I can put together some test files, if you need them, but I figured this is pretty easy to replicate.

Thanks!

**Comment by derwin12** [comment]
Yea .. I have seen that too .. source sequence has changed and now the mappings both auto and a load map dont find them.

**Comment by computergeek1507** [comment]
i think the "c" variable needs a to be lowered for the comparison, but I'm unsure of the negative repercussions..

![image](https://github.com/user-attachments/assets/d3565ae0-a3b4-44f1-ae09-b3172dfea464)

**Comment by cybercop23** [comment]
How did you create the map file in the first place? Was it from a SS sequence that had the models all in upper case?
Can you please load the xmp file?

**Comment by derwin12** [comment]
Showstoppers switch from lower to upper case in their sequences. Our map had it one way Upper? so it didnt match the lower or vica versa

**Comment by cybercop23** [comment]
> Showstoppers switch from lower to upper case in their sequences. Our map had it one way Upper? so it didnt match the lower or vica versa

Okay, that's what I thought... so need to recreate it and see.
